### PR TITLE
Add missing Gen 5 moves and fix other moves

### PR DESCRIPTION
**Pull request by AustinXII**

### CODE REVIEW

File: js/data/move_data.js [hunk 1]
```diff
@@ -1430,11 +1436,6 @@ var MOVES_BW = $.extend(true, {}, MOVES_DPP, {
         category: 'Physical',
         makesContact: true
     },
-    'Attack Order': {
```
Comment (AustinXII) [inline js/data/move_data.js]: duplicate

File: js/damage.js [hunk 2]
```diff
@@ -305,6 +305,8 @@ function getDamageResult(attacker, defender, move, field) {
         bpMods.push(0x800);
         description.moveBP = move.bp / 2;
         description.weather = field.weather;
+    } else if (defender.weight <= "200" && move.name === "Sky Drop" || move.name === "Sky Drop" && (defender.type1 === "Flying" || defender.type2 === "Flying")) {
```
Comment (Marty-D) [inline js/damage.js]: Several issues here, not least of which is this doesn't work at all.
- This should be done in `getDamageResult()` instead
- 200 shouldn't be a string
- You should be checking if the weight is >=, not less than

Basically `(move.name === "Sky Drop" && (defender.weight >= 200 || (defender.type1 === "Flying" || defender.type2 === "Flying")))` would be a better check.

File: js/data/move_data.js [hunk 3]
```diff
@@ -1494,6 +1502,13 @@ var MOVES_BW = $.extend(true, {}, MOVES_DPP, {
         category: 'Special',
         isBullet: true
     },
+    'Electro Web': {
```
Comment (Marty-D) [inline js/data/move_data.js]: Electroweb is one word.

**Comment by AustinXII** [comment]
@Marty-D updated

File: js/damage.js [hunk 4]
```diff
@@ -120,6 +120,9 @@ function getDamageResult(attacker, defender, move, field) {
     if (typeEffectiveness === 0) {
         return {"damage":[0], "description":buildDescription(description)};
     }
+    if (move.name === "Sky Drop" && (defender.weight >= 200 || (defender.type1 === "Flying" || defender.type2 === "Flying"))) {
```
Comment (Marty-D) [inline js/damage.js]: I forgot this file is also used for Gen 5 damage calcs, so this should actually be something like `(move.name === "Sky Drop" && (defender.type1 === "Flying" || defender.type2 === "Flying" || (gen >= 6 && defender.weight >= 200)))`.

**Comment by AustinXII** [comment]
done @Marty-D

**Comment by Marty-D** [comment]
:+1:

**Pull request merged**